app/main.py

```diff
@@ -1 +1,18 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    command, file_before, direction = command.split()
+    direction = direction.split("/")
```

Comment:
Is your app cross-platform?) For example, will it works with `\` on windows ?)

HINT: `use os.path.split()`